### PR TITLE
Add query for recent-active users.

### PR DESCRIPTION
**Pull request by thenav56**
Addresses
- #1048

## Changes

* Add recent active users.

Mention related users here if any.

## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [x] n+1 queries
- [x] flake8 issues
- [x] `print`
- [x] typos
- [x] unwanted comments

## This PR contains valid:

- [ ] tests
- [ ] permission checks (tests here too)
- [ ] translations

### CODE REVIEW

**Review by bimal125 — COMMENTED** [review]
Can't we get most recent active users form "last login date" and "is_active", also raw sql queries look wired.

**Comment by thenav56** [comment]
> Can't we get most recent active users form "last login date" and "is_active", also raw sql queries look wired.

Recent active users are for users under the project with their activity (i.e lead/entry created/updated).

File: apps/project/schema.py [hunk 1]
```diff
@@ -64,6 +66,28 @@
...
+    recent_active_users = [
+        (recent_active_users_map[id], date)
+        for id, date in users_activity
+        if id in recent_active_users_map
+    ]
+    return [
+        {
+            'id': user.id,
+            'name': user.get_display_name(),
+            'user_id': user.id,
```
Comment (k9845) [inline apps/project/schema.py]: Why we have `user_id` as we already have `id`?

File: utils/graphene/types.py [hunk 2]
```diff
@@ -234,3 +234,10 @@ class UserEntityCountType(graphene.ObjectType):
     name = graphene.String()
     user_id = graphene.String()
     count = graphene.Int()
+
+
+class UserEntityDateType(graphene.ObjectType):
+    id = graphene.String(required=True)
+    name = graphene.String()
+    user_id = graphene.String(required=True)
+    date = graphene.DateTime(required=True)
```
Comment (k9845) [inline utils/graphene/types.py]: What is difference between `id` and `user_id` here?

**Review by k9845 — APPROVED** [review]

**Pull request merged**